docs/ops/movement/Unique_10.md

```diff
@@ -0,0 +1,119 @@
+# Unique  {#openvino_docs_ops_comparison_Unique_10}
+
+**Versioned name**: *Unique-10*
+
+**Category**: *Data movement*
+
+**Short description**: *Unique* finds unique elements of a given input tensor. Depending on the operator's attributes it either looks for unique values globally in the whole (flattened) tensor or performs the search along the specified axis.
+
+**Detailed description**
+The operator can either work in elementwise mode searching for unique values in the whole tensor or it can consider slices of the input tensor along the specified axis. This way the op is able to find unique subtensors in the input data. Except for the unique elements the operator also produces the indices of the unique elements in the input tensor and the number of occurences of unique elements in the input data.
+
+**Attributes**:
+
+* *axis*
+
+  * **Description**: *axis* specifies the input tensor's axis along which the subtensors are processed. When the axis is set to the maximum value of the underlying integer type it means that the input tensor should be processed elementwise (the operator then looks for unique individual numbers in the whole flattened tensor).
+  * **Range of values**: The accepted range of values is `[-R, R-1]` where `R` is the rank of the input tensor.
+  * **Type**: int
+  * **Default value**: MAX_INT
+  * **Required**: *no*
+
+* *sorted*
+
+  * **Description**: controls whether the unique elements in the output tensor are sorted in ascending order.
+  * **Range of values**:
+    * false - output tensor's elements are not sorted
+    * true - output tensor's element are sorted
+  * **Type**: boolean
+  * **Default value**: true
+  * **Required**: *no*
```

Comment:
How are they ordered in case of TensorFlow and PyTorch without sort. Are they aligned with ONNX or between each other?

Reply:
PyTorch seems to sort the values by default while TF maintains the original order of the elements in the input (according to the docs):
```
>>> t
tensor([[3., 2., 1.],
        [1., 2., 3.],
        [4., 5., 6.]])
>>> torch.unique(t)
tensor([1., 2., 3., 4., 5., 6.])
>>> torch.unique(t,dim=0)
tensor([[1., 2., 3.],
        [3., 2., 1.],
        [4., 5., 6.]])
```